src/components/view/DeviceActivityView/DeviceActivityView.stories.tsx

```diff
@@ -16,4 +16,9 @@ export const Push = {
 export const Modal = {
     args: { presentation: "Modal", previewState: "default" },
     render: render
+};
+
+export const Live = {
+    args: { previewState: "live" },
```

Comment:
I don't think this is correct here.  This view only supports a preview state of `"default" | undefined`, so if you want a live state, I think you can just omit the preview state.